lorien/InfiniteCanvas/Tools/CanvasTool.gd

```diff
@@ -89,6 +102,9 @@ func end_stroke() -> void:
 	_canvas.end_stroke()
 	performing_stroke = false
 
+func eval_disable_stroke() -> void:
```

Comment:
I probably would not have made a separate method for a oneliner like that. Anyway this should have an underscore prefix to indicate it's a private method.

Reply:
A rule of thumb I use in OOP is: "if some specific line(s) of code are required more than once, create a separate function for it". Increases modularity and makes it easier to maintain, since you don't need to think about where you used that specific expression elsewhere as it's already encapuslated in a separate function/method. There are reasonable exceptions of course.

GDScript doesn't have the concept of private members. But sure, to signal the intended use, I prepended an underscore now. 🙂

Let me know if you would like to omit a separate method for that oneliner and I'll change that then as well.

Reply:
Yeah GDscript doesn't have private members, but a common convention is to prefix them with an underscore to indicate "privateness" :smile: